File: bot/pages/admin_panel.py
from telegram import InlineKeyboardMarkup, InlineKeyboardButton
from telegram.ext import ContextTypes

def create_message(context: ContextTypes.DEFAULT_TYPE) -> dict:
    buttons = [
        [InlineKeyboardButton(text=context._chat_data.get_lang()['button.admin.clear_expired_cache'], callback_data='admin.clear_expired_cache')],
        [InlineKeyboardButton(text=context._chat_data.get_lang()['button.admin.clear_all_cache'], callback_data='admin.clear_all_cache')],
        [InlineKeyboardButton(text=context._chat_data.get_lang()['button.admin.get_logs'], callback_data='admin.get_logs')],
        [InlineKeyboardButton(text=context._chat_data.get_lang()['button.admin.clear_logs'], callback_data='admin.clear_logs')],
        [InlineKeyboardButton(text=context._chat_data.get_lang()['button.back'], callback_data='open.menu')]
    ]

    return {
        'text': context._chat_data.get_lang()['page.admin_panel'],
        'reply_markup': InlineKeyboardMarkup(buttons),
        'parse_mode': 'MarkdownV2'
    }

File: internal/dteubot/pages/admin_panel.go
package pages

import (
	"github.com/cubicbyte/dteubot/internal/data"
	"github.com/cubicbyte/dteubot/internal/dteubot/utils"
	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
)

func CreateAdminPanelPage(cm *data.ChatDataManager) (*Page, error) {
	lang, err := utils.GetLang(cm)
	if err != nil {
		return nil, err
	}

	page := Page{
		Text: lang.Page.AdminPanel,
		InlineKeyboard: tgbotapi.NewInlineKeyboardMarkup(
			tgbotapi.NewInlineKeyboardRow(
				tgbotapi.NewInlineKeyboardButtonData(lang.Button.ClearCache, "admin.clear_cache"),
			),
			tgbotapi.NewInlineKeyboardRow(
				tgbotapi.NewInlineKeyboardButtonData(lang.Button.ClearLogs, "admin.clear_logs"),
			),
			tgbotapi.NewInlineKeyboardRow(
				tgbotapi.NewInlineKeyboardButtonData(lang.Button.GetLogs, "admin.get_logs"),
			),
			tgbotapi.NewInlineKeyboardRow(
				tgbotapi.NewInlineKeyboardButtonData(lang.Button.Back, "open.menu"),
			),
		),
		ParseMode: "MarkdownV2",
	}

	return &page, nil
}